pkg/cmd/roachtest/cluster.go

```diff
@@ -174,6 +174,7 @@ func findBinaryOrLibrary(binOrLib string, name string) (string, error) {
 		dirs := []string{
 			filepath.Join(gopath, "/src/github.com/cockroachdb/cockroach/"),
 			filepath.Join(gopath, "/src/github.com/cockroachdb/cockroach", binOrLib+suffix),
+			filepath.Join(gopath, "/src/github.com/cockroachdb/cockroach", binOrLib),
```

Comment:
Will this have unintended side effects? For instance, what happens if there are both local and remote (cross-compiled) versions of the file being searched for?

Reply:
I moved this line to the end of the list, so now this won't change the behavior of anything that was previously working.